grpc/tracing.go

```diff
@@ -1,38 +1,37 @@
 package grpc
 
 import (
-	"github.com/contiamo/goserver"
+	"github.com/contiamo/go-base/v2/pkg/tracing"
 	"github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc"
 	opentracing "github.com/opentracing/opentracing-go"
 	"google.golang.org/grpc"
 )
 
 // WithTracing configures the server to support tracing in the opentracing format.
-// It attaches a root span to the context of all incoming requests and configures the tracer to send the traces to the configured opentracing server.
-func WithTracing(server, serviceName string) Option {
-	return &tracingOption{server, serviceName}
+// It attaches a root span to the context of all incoming requests.
+//
+// The tracer is configured via the JAEGER_* env variables.
+//
+// The server variable is deprecated and will be ignored.
+func WithTracing(serverDeprecated, serviceName string) Option {
```

Comment:
This aligns with the http middelware in go-base https://github.com/contiamo/go-base/blob/master/pkg/http/middlewares/tracing.go#L17-L25